### ISSUE TITLE
definitions should not end in "proofbox"

### ISSUE THREAD

**Issue by kohlhase** [MathHubInfo/Frontend#127]
currently, all definitions, e.g. in [functions.en](https://beta.mathhub.info/library/document/cHNldWRvLXRyZWU6Ly9zbWdsb20tc3RleC9zbWdsb20vc2V0cy9mdW5jdGlvbnMvZW4=) end in a proof box. 
I suspect that is an artefact of the CSS beautification @Jazzpirate added to LaTeXML generation.

**Comment by Jazzpirate** [comment]
No, this is literally what latexml outputs... afaik, LaTeXML doesn't like redefinitions of environments such as `definition` etc and puts its own output-spin on it either way

**Comment by kohlhase** [comment]
Then that is something we will have to look into. We could just use `\begin/\end"\{sdefinition}` everywhere, and deprecate `\begin/\end"\{definition}`.
